pkg/config/config.go

```diff
@@ -229,7 +227,6 @@ func (cd *configuration) Load(cm *corev1.ConfigMap) {
 	}
 	if reconcilePolicyReport {
 		logger.Info("resource filters changed, sending reconcile signal to the policy controller")
-		cd.reconcilePolicyReport <- true
```

Comment:
How do we update reports on configmap changes?

Reply:
Maybe we can remove the entire if block.

Reply:
Done